team_color : 'yellow'
team_side  : 'left'
